src/openvino-ir.js

```diff
@@ -0,0 +1,656 @@
+/*jshint esversion: 6 */
+
+var openvinoIR = openvinoIR || {};
+
+openvinoIR.ModelFactory = class {
+    match(context) {
+        return context.identifier.endsWith('.xml') || context.identifier.endsWith('.dot');
+    }
+
+    open(context, host, callback) {
+        host.require('./openvino-ir-parser', (err) => {
```

Comment:
Have a look at recent `tflite.js` for similar pattern. Attaching to `window` will not work in test environment.

```
host.require('./openvino-parser', (err, openvino_parser) => {
   var x = new openvino_parser.OpenVINOIRParser(...);
});

Reply:
I have started refactoring on my branch in the fork `feature/demidovs/fix-e2e-test`, however, when I run `npm run test openvino`, I get the following error:

```sh
events.js:307
    throw new TypeError('"listener" argument must be a function');
    ^

TypeError: "listener" argument must be a function
```

Reply:
@lutzroeder is it a known problem? I am now trying to rootcause it also